api/uninstall/service.go

```diff
@@ -3,24 +3,34 @@ package uninstall
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/gojekfarm/albatross/pkg/helmcli"
 	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
 
 	"helm.sh/helm/v3/pkg/release"
 )
 
+const defaultTimeout = 5 * time.Minute
```

Comment:
would be good to stick to seconds here also because [here](https://github.com/gojekfarm/albatross/pull/8/files#diff-abe488ec79d7297891bdbf680a982e7c0e9f2286b39dd6478efe0d8084176e44R26) it says seconds. Lets keep it uniform

Reply:
Kept it time.Minute since it's easier to use.
the unit is different here though, the input we take is an int which is time in seconds. whereas default Timeout is in time.Duration.

Reply:
@punit-kulal frankly it is going to be readable anyways because you're going to mention `time.Second` and secondly it's usually easier to stick to one unit think of it from reader's perspective I feel sticking to same unit makes sense and we know usually timeouts are mentioned in ms or seconds

Reply:
updated to seconds